openmdao/utils/general_utils.py

```diff
@@ -410,3 +411,28 @@ def run_model(prob):
         sys.stdout = stdout
 
     return strout.getvalue()
+
+@contextlib.contextmanager
+def printoptions(*args, **kwargs):
+    """Context manager for setting print options.
+    Set print options for the scope of the `with` block, and restore the old
+    options at the end. See `set_printoptions` for the full description of
+    available options.
+    Examples
+    --------
+    >>> with np.printoptions(precision=2):
```

Comment:
no `np.` for the context...

Reply:
should we put the version check in here?  and automatically conform to 1.13 formatting?

Reply:
If you do make a change for this...can you add the context manager to the test I originally did this for?

Reply:
I'd rather not bury any hidden formatting stuff in there since this is a general function and will probably eventually go away and be replaced with the verison inside of numpy (which is identical to what we have)